dicom_standard/process_ciod_macro_relationship.py

```diff
@@ -0,0 +1,35 @@
+'''
+Takes the extracted CIOD-Macro table information to build a list of all
+CIOD-Macro relationships defined in the DICOM Standard.
```

Comment:
See earlier comment about naming here. I think it would be useful to be explicit that this is the CIOD-Functional group macro relationship, rather than a generic listing of all macros within a CIOD.